### ISSUE TITLE
'Template' object has no attribute 'nodelist'

### ISSUE THREAD

**Issue by comzeradd** [jdriscoll/django-photologue#132]
I'm getting this error on `root.html` template.

Traceback:

```
Template error:
In template /usr/local/lib/python2.7/dist-packages/photologue/templates/photologue/root.html, error at line 1
   'Template' object has no attribute 'nodelist'
   1 :  {% extends "base.html" %} 


   2 : 

Traceback:
File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py" in get_response
  137.                 response = response.render()
File "/usr/local/lib/python2.7/dist-packages/django/template/response.py" in render
  103.             self.content = self.rendered_content
File "/usr/local/lib/python2.7/dist-packages/django/template/response.py" in rendered_content
  80.         content = template.render(context)
File "/usr/local/lib/python2.7/dist-packages/django/template/base.py" in render
  148.             return self._render(context)
File "/usr/local/lib/python2.7/dist-packages/django/template/base.py" in _render
  142.         return self.nodelist.render(context)
File "/usr/local/lib/python2.7/dist-packages/django/template/base.py" in render
  844.                 bit = self.render_node(node, context)
File "/usr/local/lib/python2.7/dist-packages/django/template/debug.py" in render_node
  80.             return node.render(context)
File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py" in render
  126.         return compiled_parent._render(context)
File "/usr/local/lib/python2.7/dist-packages/django/template/base.py" in _render
  142.         return self.nodelist.render(context)
File "/usr/local/lib/python2.7/dist-packages/django/template/base.py" in render
  844.                 bit = self.render_node(node, context)
File "/usr/local/lib/python2.7/dist-packages/django/template/debug.py" in render_node
  80.             return node.render(context)
File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py" in render
  115.         for node in compiled_parent.nodelist:

Exception Type: AttributeError at /fxosmedia/photologue/gallerylist/
Exception Value: 'Template' object has no attribute 'nodelist'
```

**Comment by richardbarran** [comment]
Hi,
Are you using Django 1.8 with Jinja2 templates?

**Comment by comzeradd** [comment]
Yes :) Jinja2 with jingo specifically.

This happens when I exclude photologue from jinja rendering.

**Comment by richardbarran** [comment]
So you're using Jinja2 for your project, except for Photologue, which uses Django's standard template system?
Could you post your ``TEMPLATES`` setting please? I'm still wrapping my head around this new feature of Django 1.8 :-)

**Comment by comzeradd** [comment]
My template settings pretty much summarize to this:

```
TEMPLATE_LOADERS = (
    'jingo.Loader',
    'django.template.loaders.filesystem.Loader',
    'django.template.loaders.app_directories.Loader',
)

JINGO_EXCLUDE_APPS = ['browserid', 'admin', 'photologue']
```

**Comment by richardbarran** [comment]
I *think* that you don't need Jingo any more with Django 1.8, and that Jinja2 templates work out of the box.
In the docs at https://docs.djangoproject.com/en/1.8/topics/templates/ I found this snippet:

```
TEMPLATES = [
    {
        'BACKEND': 'django.template.backends.django.DjangoTemplates',
        'DIRS': [
            '/home/html/example.com',
            '/home/html/default',
        ],
    },
    {
        'BACKEND': 'django.template.backends.jinja2.Jinja2',
        'DIRS': [
            '/home/html/jinja2',
        ],
    },
]
```
Disclaimer: I have not used Django 1.8 yet!

**Comment by richardbarran** [comment]
Hi @comzeradd - did you fix your issue?

**Comment by comzeradd** [comment]
Well, I eventually figured an altertive way to do it. But yes, regrading this issue it can be closed, since 1.8 seems to work nice with Jinja2.

Thnaks
